rules/signing.bzl

```diff
@@ -948,3 +948,82 @@ keyset = rule(
         ),
     },
 )
+
+def _signature_test_impl(ctx):
+    keys = []
+    tc = ctx.toolchains[LOCALTOOLS_TOOLCHAIN]
+    script = ctx.actions.declare_file("{}.bash".format(ctx.label.name))
+    verify_args = ""
+    ecdsa_key = key_from_dict(ctx.attr.ecdsa_key, "ecdsa_key")
+    spx_key = key_from_dict(ctx.attr.spx_key, "spx_key")
+
+    selected_ecdsa_key_path = ""
+    selected_spx_key_path = ""
+
+    if ctx.attr.spx_domain == "":
+        print("WARNING: Using default test parameters")
```

Comment:
We probably should eliminate this `print`, as this is likely to emit a bunch of prints into the output regardless of whether we're executing this test (the prints would happen during the bazel analysis phase, which happens whenever you run a bazel command).